common/connection.go

```diff
@@ -378,20 +379,17 @@ func (c *Connection) send(ctx context.Context, msg *cdproto.Message, recvCh chan
 	case c.sendCh <- msg:
 	case err := <-c.errorCh:
 		c.logger.Debugf("Connection:send:<-c.errorCh", "wsURL:%q sid:%v, err:%v", c.wsURL, msg.SessionID, err)
-		return err
+		return fmt.Errorf("sending message to browser over WebSocket: %w", err)
 	case code := <-c.closeCh:
 		c.logger.Debugf("Connection:send:<-c.closeCh", "wsURL:%q sid:%v, websocket code:%v", c.wsURL, msg.SessionID, code)
 		_ = c.closeConnection(code)
-		return &websocket.CloseError{Code: code}
+		return fmt.Errorf("closing the WebSocket communication with the browser: %w", &websocket.CloseError{Code: code})
+	case <-ctx.Done():
+		c.logger.Debugf("Connection:send:<-ctx.Done", "wsURL:%q sid:%v err:%v", c.wsURL, msg.SessionID, c.ctx.Err())
+		return nil
```

Comment:
This will probably lift some errors. There is no need for an error after closing.

Reply:
OK, let's leave `Debugf` logs as is then in this PR.

But I think we should at least standardize the category regardless of the log level we're using. Putting method names in the category field seems wrong, and we'll probably need to standardize it for #416.